<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinDialog</class>
 <widget class="QDialog" name="JoinDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join a Session</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="newVersionNotification">
     <property name="styleSheet">
      <string notr="true">background: #cbe3ef;
color: #000;
border: 1px solid #3daee9;
border-radius: 3px;
padding: 6px;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>You are running an outdated version of Drawpile. Some servers will be unavailable.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="noListServersNotification">
     <property name="styleSheet">
      <string notr="true">background: #cbe3ef;
color: #000;
border: 1px solid #3daee9;
border-radius: 3px;
padding: 6px;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="text">
      <string><![CDATA[<p>You havenâ€™t added any servers yet. Find some at <a href="https://drawpile.net/communities/">drawpile.net/communities</a>, or try pub.drawpile.net.</p>]]></string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="filter">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Show:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showClosed">
       <property name="text">
        <string>Closed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showPassworded">
       <property name="text">
        <string>Passworded</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showNsfw">
       <property name="text">
        <string>NSFW</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="listing">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Enter a host address, session URL, or room code to join a session:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QComboBox" name="address">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="mandatoryfield" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="autoRecord">
       <property name="toolTip">
        <string>Record session</string>
       </property>
       <property name="icon">
        <iconset theme="media-record">
         <normaloff>theme:media-record.svg</normaloff>theme:media-record.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>address</tabstop>
  <tabstop>buttons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>JoinDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>JoinDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
